src/react.ts

```diff
@@ -3,10 +3,10 @@ import { useState } from 'react';
 import useDeepCompareEffect from 'use-deep-compare-effect';
 
 import { ContentfulLivePreview } from '.';
-import { Entity } from './types';
+import { Argument } from './types';
 import { debounce } from './utils';
 
-export function useContentfulLiveUpdates<T extends Entity | null | undefined>(
+export function useContentfulLiveUpdates<T extends Argument | null | undefined>(
   data: T,
```

Comment:
how about `data: T = {}` to fallback to an empty object?

I could be wrong but
```
}, [data || {}]);
```

looks like it's creating a new object on every render

Reply:
Sadly not working, because default parameters only are used on `undefined` but not for null :( 
but we can use `useDeepCompareEffectNoCheck`